src/libraries/Common/tests/TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -301,6 +304,21 @@ private static bool IsDistroAndVersionOrHigher(Predicate<string> distroPredicate
             return false;
         }
 
+        private static bool IsQemuLinux()
+        {
+            if (IsLinux)
+            {
+                foreach (DictionaryEntry entry in Environment.GetEnvironmentVariables())
+                {
+                    if (((String)entry.Key).ToUpper().Contains("QEMU"))
```

Comment:
This is casting a broad net. Can this only check for one or a few specific environment variables?

Reply:
I can search for _QEMU_LD_PREFIX_ we set on Docker, _DOTNET_RUNNING_UNDER_QEMU_ you proposed or something else. I thought whatever containing "QEMU" would be fine.

Reply:
> whatever containing "QEMU" would be fine.

I am worried about false positives that this can produce.

> DOTNET_RUNNING_UNDER_QEMU 

This would be my preference.